packages/devtools/cli/src/commands/agent/stop.ts

```diff
@@ -13,6 +13,7 @@ export default class Stop extends BaseCommand<typeof Stop> {
   static override flags = {
     ...BaseCommand.flags,
     all: Flags.boolean({ char: 'a', description: 'Stop all agents.' }),
+    force: Flags.boolean({ char: 'f', description: 'Force stop.' }),
```

Comment:
don't create single letter abbreviations by default -- and not for important "are you sure" kind of things.